PowerShell/BuildingBlocks/ServicesAndTasks/ExportRunbook.ps1

```diff
@@ -1,12 +1,10 @@
-param(
-    [Parameter(Mandatory=$true)]
-    [guid]$OfferingMetadataId,
+# Initialize variables
+$offeringMetadataId = [GUID](Read-Host -Prompt 'Runbook ID')
+$folderPath = Read-Host -Prompt 'Folder path to export to'
 
-    [Parameter(Mandatory=$true)]
-    [string]$FolderPath)
+# Authenticate
```

Comment:
Same question about the order in which information is requested here.

Reply:
Fixed